Aztec/Classes/Formatters/HeaderFormatter.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import UIKit
+
+class HeaderFormatter: ParagraphAttributeFormatter {
+
+    enum HeaderType: Int {
+        case none = 0
+        case h1 = 1
+        case h2 = 2
+        case h3 = 3
+        case h4 = 4
+        case h5 = 5
+        case h6 = 6
+
+        var fontSize: CGFloat {
+            switch self {
+            case .none: return 14
+            case .h1: return 36
+            case .h2: return 24
+            case .h3: return 21
+            case .h4: return 16
+            case .h5: return 14
+            case .h6: return 11
+            }
+        }
+    }
+
+    let elementType: Libxml2.StandardElementType = .h1
+
+    var headerLevel: HeaderType = .h1
```

Comment:
A few changes here:

1. Change `var` to `let`.
2. Remove the `= .h1` assignment, since the initializer is taking care of it.